cmd/ec2.go

```diff
@@ -15,21 +21,45 @@ var ec2Cmd = &cobra.Command{
 	Short: "Print ec2 related information",
 	Long: `For example,
 		awsctl get ec2`,
-	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("ec2 called")
-	},
+	Run: getEC2command,
 }
 
-func init() {
-	getCmd.AddCommand(ec2Cmd)
+func getEC2command(cmd *cobra.Command, args []string) {
+
+	ctx := context.TODO()
+	cfg, err := config.LoadDefaultConfig(ctx)
+	if err != nil {
+		log.Fatal("Config Error Occured", err)
+	}
+
+	client := ec2.NewFromConfig(cfg)
+	
+
+	input := &ec2.DescribeInstancesInput{}
 
-	// Here you will define your flags and configuration settings.
+	info, err := pkg.GetEc2Info(ctx, input, client)
```

Comment:
instead of passing input, can we pass generic information and then construct this input inside the pkg/ functions?

Reply:
Will try.